lib/react_client.dart

```diff
@@ -429,7 +429,7 @@ _convertEventHandlers(Map args) {
   args.forEach((propKey, value) {
     var eventFactory = _eventPropKeyToEventFactory[propKey];
     if (eventFactory != null && value != null) {
-      args[propKey] = (events.SyntheticEvent e, [String domId, Event event]) => zone.run(() {
+      args[propKey] = (events.SyntheticEvent e, [_, __]) => zone.run(() {
```

Comment:
The reason is because now the second arg passed to this is now of type `EventImpl`. And because we really don't care what it is we can just ignore it.